trajectory_publisher_node:
  ros__parameters:
    trajectory_generator_name: sanding_generator
    update_frequency: 100.0
    n_buffer_points: 5
    frame_of_interest: ati_mini45_measurement_reference
    initial_targets:
      robot_configuration: [0.1865, 0.0, -2.4, 0.0, 2.5915]
      robot_velocity: [0.0, 0.0, 0.0, 0.0, 0.0]
      frame_translation: [0.5, 0.0, 0.4]
      frame_rotation: [0.0, 0.0, 1.0, 0.0]
      desired_force: [0.0, 0.0, 5.0, 0.0, 0.0, 0.0]
    weights:
      robot_configuration: [0.0001, 0.0001, 0.0001, 0.0001, 0.0001]
      robot_velocity: [0.001, 0.001, 0.001, 0.001, 0.001]
      robot_effort: [0.001, 0.001, 0.001, 0.001, 0.001]
      frame_rotation: [5.0, 5.0, 0.0]
      frame_translation: [30.0, 30.0, 1.0]
      frame_angular_velocity: [0.0, 0.0, 0.0]
      frame_linear_velocity: [2.0, 2.0, 0.0]
      desired_force: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0] # 0.001
    sanding_generator:
      circle:
        radius: 0.05
        frequency: 0.1
      start_time: 5.0
